src/clj/rems/api/schema.clj

```diff
@@ -6,17 +6,30 @@
             [schema.core :as s])
   (:import (org.joda.time DateTime)))
 
+(declare Organization)
 (def UserId s/Str)
-
 (s/defschema User {:userid UserId})
+(s/defschema OrganizationId {:organization/id s/Str})
 
 (s/defschema UserWithAttributes
   {:userid UserId
    :name (s/maybe s/Str)
    :email (s/maybe s/Str)
-   (s/optional-key :organization) (s/maybe s/Str)
+   (s/optional-key :organizations) [OrganizationId]
```

Comment:
could the key be made non-optional? default to `[]`? might simplify some code.

Reply:
I think I put it originally like this because we need the migration for existing user data, and make sure this key is always saved in the future. Since those parts are not yet there this goes with optional. I'll add that as a note for a separate PR (where the parsing will be).

Reply:
As discussed I will leave this for a follow-up PR as it seems to be a big change (test expectations). Found in the ticket.